dm/tests/openapi/client/openapi_task_check

```diff
@@ -176,6 +176,72 @@ def get_task_list(task_count):
     assert data["total"] == int(task_count)
 
 
+def pause_task_success(task_name, source_name):
+    url = API_ENDPOINT + "/" + task_name + "/pause"
+    resp = requests.post(
+        url=url,
+        json=[
+            source_name,
+        ],
+    )
+    assert resp.status_code == 200
+
+
+def resume_task_success(task_name, source_name):
+    url = API_ENDPOINT + "/" + task_name + "/resume"
+    resp = requests.post(
+        url=url,
+        json=[
+            source_name,
+        ],
+    )
+    assert resp.status_code == 200
+
+
+def operate_schema_and_table_success(task_name, source_name):
+    schema_url = API_ENDPOINT + "/" + task_name + "/sources/" + source_name + "/schemas"
+    resp = requests.get(url=schema_url)
+    assert resp.status_code == 200
+    print("get_task_schema_success schema resp=", resp.json())
+    assert len(resp.json()) > 0
```

Comment:
Do we need to check the content here?

Reply:
add more check in https://github.com/pingcap/ticdc/pull/3171/commits/b4575a1b56c20ad7553fd61779c6534a4398dea2